arcgis-ios-sdk-samples/Layers/Identify KML features/IdentifyKMLFeaturesViewController.swift

```diff
@@ -36,10 +36,9 @@ class IdentifyKMLFeaturesViewController: UIViewController {
         let map = AGSMap(basemapStyle: .arcGISDarkGrayBase)
         map.operationalLayers.add(forecastLayer)
         
-        let center = AGSPoint(x: -48_885, y: 1_718_235, spatialReference: AGSSpatialReference(wkid: 5070))
-        map.initialViewpoint = AGSViewpoint(center: center, scale: 50_000_000)
-        
         mapView.map = map
+        let center = AGSPoint(x: -48_885, y: 1_718_235, spatialReference: AGSSpatialReference(wkid: 5070))
```

Comment:
Thank you for moving this variable. In some cases, due to the creation of the viewpoint moving, the variable for the point used as the center of the viewpoint is separated from creating and setting the viewpoint itself.